terraform/modules/kubernetes/kubernetes.tf

```diff
@@ -12,10 +12,8 @@ variable "container_registry" {
 }
 
 variable "workflow_manager_image" {
-  type = string
-  # This should be "prio-data-share-processor" but we have not yet renamed the
-  # container image
-  default = "prio-workflow-manager"
+  type    = string
+  default = "letsencrypt"
```

Comment:
I get why you deleted the obsolete comment, but `letsencrypt` is wrong, and [`prio-workflow-manager`](https://hub.docker.com/repository/docker/letsencrypt/prio-workflow-manager) was correct.